examples/quickstart/README.md

```diff
@@ -112,12 +26,7 @@ You can use Google Colab to see ZenML in action, no signup / installation requir
 
 ## :computer: Run Locally
 
-Note: Please note that the current version of the MLflow quickstart requires
-Python version 3.8 or higher. If you are using a lower version, we recommend
-that you use the python3.7 version of the quickstart instead. Please note that
-the MLflow model registry is not supported in the python3.7 version or lower.
-
-### :page_facing_up: Prerequisites 
+### :page_facing_up: Prerequisites
 To run locally, install ZenML and pull this quickstart:
```

Comment:
Not really sure what the instruction should be changed to. We haven't removed
the examples CLI command yet, nor have we removed / updated the examples
directory yet etc. I think the default would be to leave this as it is until
those other two pieces change. There will in any case be an effort to remove
those comments from READMEs etc later, no? WDYT?

Reply:
I don't think there will be an such an effort since we're removing all other examples anyways, so only quickstart needs to be adjusted actually. I would just say "clone ZenML and go into examples/quickstart" instead of the `zenml example ...` instructions.